packages/manager/modules/anthos/src/anthos-tenants.service.js

```diff
@@ -38,16 +38,33 @@ export default class AnthosTenantsService {
       {
         headers: {
           Pragma: 'no-cache',
-          'x-pagination-mode': 'CachedObjectList-Pages',
-          'x-pagination-number': pageNumber,
-          'x-pagination-size': pageSize,
+          'X-Pagination-Mode': 'CachedObjectList-Pages',
+          'X-Pagination-Number': pageNumber,
+          'X-Pagination-Size': pageSize,
           ...(typeof additional === 'undefined'
             ? {}
             : {
-                'x-pagination-filter': `additional:eq=${additional}`,
+                'X-Pagination-Filter': `additional:eq=${additional}`,
               }),
         },
       },
     );
   }
+
+  getStorageVolumes(serviceName) {
+    return this.$http
+      .get(`/dedicated/anthos/tenants/${serviceName}/storage/netapp/svms`, {
+        headers: {
+          'X-Pagination-Mode': 'CachedObjectList-Pages',
+          'X-Pagination-Size': 50000, // Expected to get only around 1-50 records
```

Comment:
```suggestion
          'X-Pagination-Size': 50, // Expected to get only around 1-50 records
```
If expected maximum is only 50 then we could restrict no ?

Reply:
Yes... But I guess we can be safe by setting it at a high number... What if there are 51 records? This would then fail...
50 is just approximate. It is not a hard boundary